File: 2nd year/MPP/project-mpp/java-project/TriathlonServices/src/main/java/triathlon/services/TriathlonException.java
package triathlon.services;

public class TriathlonException extends Exception {
    public TriathlonException() {
    }

    public TriathlonException(String message) {
        super(message);
    }

    public TriathlonException(String message, Throwable cause) {
        super(message, cause);
    }

}

File: 2nd year/MPP/project-mpp/csharp-project/services/TriathlonException.cs
using System;

namespace services
{
    public class TriathlonException : Exception
    {
        public TriathlonException() : base() { }

        public TriathlonException(string msg) : base(msg) { }

        public TriathlonException(string msg, Exception ex) : base(msg, ex) { }
    }
}
